# Cartesian 6DOF Stiffness for pose impedance control
# cartesian_stiffness_target: [600, 600, 600, 50, 50, 50] # Max values from franka example controllers (For good tracking with 200 integration gain)
cartesian_stiffness_target: [300, 300, 300, 50, 50, 50] # Follows target and is somewhat compliant at the same time 
# cartesian_stiffness_target: [50.0, 1.0, 1.0, 50, 50, 50] # This yields gravity compensation basically -- need to test on real robot

# Nominal nullspace stiffness used for all joints
# nullspace_stiffness: 0.01 #Used in LTL-RSS DEMO
nullspace_stiffness: 0.1 #Used in MIT MUSEUM DEMO

# If leave commented, then will use q_initial as q_d_nullspace_
q_nullspace: [0.0, -0.1516284550464292, 0.0, -2.1602991589328693, 0.0063609233109487425, 2.030401506252017, 0.8428664707532099]

# F_ext wrench to compensate for camera or tool on end-effector
external_tool_compensation: [0, 0, 0.0, 0, 0.0, 0.0] #Check F_ext



#### OLD VALUES FROM LTLDS-RSS PAPER (Keeping them here for historical reasons)!!
# RSS: teach, can only move along y,z or rotate around y:
# cartesian_stiffness_target: [1000, 0, 0, 50, 0, 50]
# RSS: replay:
# cartesian_stiffness_target: [600, 600, 500, 50, 50, 50]
# cartesian_stiffness_target: [300, 300, 300, 50, 50, 50]
# cartesian_stiffness_target: [1000, 200, 200, 50, 50, 50]



